### DISCUSSION TITLE
How uninstall vs code in android.

### DISCUSSION THREAD

**Discussion by Priyanshuadeveloper** [coder/code-server#4426] [Q&A] [answered]
If anyone knows how to uninstall vs code in android. Then please tell. I have install it by doing these steps:- https://www.codewithharry.com/blogpost/install-vs-code-in-android

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Try this: `apt remove code-server`, it might work.

**Reply by Priyanshuadeveloper** [reply]
![Screenshot_2021-11-02-16-47-31-45_84d3000e3f4017145260f7618db1d683](https://user-images.githubusercontent.com/89924793/139837111-4c676152-c45c-461d-90cc-72526fab75dc.jpg)
Bro it is showing this answer, is really vs code is unistalled?

**Reply by lg-nineoneninethreenine** [reply]
Dangit! @jsjoeio please help here

**Reply by jsjoeio** [reply]
Hmm... can you try typing `which code-server`? I think that should print the path where the binary is installed. cc @code-asher

**Reply by code-asher** [reply]
According to the linked blog post they are just downloading the tar and extracting it in the current directory so removing the directory would be the way to go.  For example `rm code-server-3.10.2-linux-arm64 -r`.

**Reply by Priyanshuadeveloper** [reply]
@code-asher dude the result which termux shows me is :- rm: cannot remove 'code-server-3.10.2-linux-arm64': No such file or directory

**Reply by lg-nineoneninethreenine** [reply]
That was an example!

**Reply by lg-nineoneninethreenine** [reply]
Instead of 3.10.2, put your present version.

**Reply by Priyanshuadeveloper** [reply]
@im-coder-lg but my code server version is also 3.10.2 only.
And I want to ask you that if I delete temux from my phone then it's all data with vs code and ubuntu, all will delete?

**Reply by lg-nineoneninethreenine** [reply]
Yes, everything will delete.

**Reply by Priyanshuadeveloper** [reply]
Thanks it is deleted.
